app/controllers/users.js

```diff
@@ -1,65 +1,61 @@
-/**
- * Module dependencies.
- */
-var mongoose = require('mongoose'),
-  User = mongoose.model('User');
-var avatars = require('./avatars').all();
+/* global next:true*/
+/* eslint no-undef: "error"*/
+/* eslint no-underscore-dangle: ["error", { "allow": ["_id"]}]*/
+/* eslint no-unused-vars: ["error", { "argsIgnorePattern": "^_" }]*/
```

Comment:
The _id and 'next' variables are in use in different files that we are not working on currently and touching them crashes the application, making it hard for us to test the functionality of the feature.